doc/dev/MAINTAINERS_AND_CONTRIBUTORS.md

```diff
@@ -21,6 +21,7 @@ Contributors with a single contribution or that may not want to are not listed h
 | -------------- | ---------- | ---------- |
 | jrushlow       | 19jrushlow | Yes        |
 | ssmncrf        | samoncrief | Yes        |
+| Evgenii Startcev        | exbluesbreaker | Yes        |
```

Comment:
Could you add your discord alias on F3D discord as well, something like : `Evgenii Startcev (PhilHellmut)`. The point is about attribution but also being able to find people more easily on discord :)